### DISCUSSION TITLE
Get line numbers in Prose Code

### DISCUSSION THREAD

**Discussion by dbzx10299** [nuxt/content#1970] [Q&A]
How can we modify the ProseCode component to output line numbers?

**Comment by nobkd** [comment]
Because #1973 was merged, you can use edge channel (until the next release) to add line numbers by using css `attr(line)` to access the line attribute of code block lines in `content` property when styling code block lines with `::before`
https://github.com/nuxt/content/pull/1973#issuecomment-1485268998
